packages/forms/backend/sirius-components-collaborative-forms/src/main/resources/schema/form.graphqls

```diff
@@ -77,6 +77,11 @@ type ContainerBorderStyle {
   radius: Int!
   size: Int!
   lineStyle: ContainerBorderLineStyle!
+  justifyContent: ContainerContentAlignment!
```

Comment:
Why does any of this work has to be on a ContainerBorderStyle?

Reply:
properties are not link to border anymore